packages/ui/src/components/asset-name/asset-name.module.css

```
@@ -0,0 +1,32 @@
+.root {}
```

Biome

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)